packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "eslint-plugin-codemod",
+  "version": "0.0.1",
+  "dependencies": {
+    "eslint-codemod-utils": "latest"
```

Comment:
Was `latest` intentional? Would probably be better to specify the version here unless there's a reason that i'm missing 🤔

Reply:
I mean it was a bit meh. I'll add a caret range.